test/TensorFlowRuntime/dynamic_attributes.swift

```diff
@@ -259,9 +259,10 @@ DynamicAttributeTests.testAllBackends("NormalAttribute Array<Bool>") {
 }
 
 DynamicAttributeTests.testAllBackends("NormalAttribute Array<Int32>") {
-  let result = convImage.convolved2D(withFilter: convFilter,
-                                     strides: loadStridesInt32(),
-                                     padding: .valid)
+  let result: Tensor<Float> = #tfop("Conv2D", convImage, convFilter,
```

Comment:
I don't see why this test needs to be changed. If there is a reason, it should use `Raw` instead of `#tfop`.

Reply:
I changed it to be consistent with `loadStridesInt64()` and the `NormalAttribute Array<Int64>` test below.
My understanding is that `dynamic_attributes.swift` is designed to specifically test `#tfop`, not `Raw`.